examples/template-svelte-js/src/service-worker.js

```diff
@@ -0,0 +1,51 @@
+const CACHE_NAME = `cache_${TIMESTAMP}`;
```

Comment:
Is the service worker necessary as well? For simple examples, it might be unnecessary.

Reply:
No, it is not necessary. I can remove a couple of files (manifest, robots, service-worker, logos) to make the template smaller.

Reply:
Yes please!

Reply:
Done!